app.js

```diff
@@ -1,8 +1,8 @@
 const express = require("express");
 const app = express();
 const {getTopics} = require ("./contorllors/getTopics");
-const {getArticles, getArticleById, getArticleCommentsById} = require ("./contorllors/articles");
-const { postComment } = require("./contorllors/articles");
+const {getArticles, getArticleById, getArticleCommentsById, postComment, patchVotes} = require ("./contorllors/articles");
```

Comment:
just as a headsup that there was a spelling error when creating the controllers folder - as long as it's consistent like here when requiring in, it's technically fine though